homeassistant/components/websocket_api/messages.py

```diff
@@ -104,7 +104,7 @@ def error_message(
     }
 
 
-def event_message(iden: int, event: Any) -> dict[str, Any]:
+def event_message(iden: int | str, event: Any) -> dict[str, Any]:
```

Comment:
This is changing a fundamental model that is used across all of Home Assistant. We cannot make this change. It's defined all the way here https://github.com/home-assistant/home-assistant-js-websocket/blob/master/lib/types.ts#L8-L9

Reply:
This change is not needed anymore.